qualang_tools/bakery/readme.md

```diff
@@ -0,0 +1,273 @@
+# Waveform baking
+This library introduces a new framework for creating arbitrary waveforms and storing them in a usual configuration file. 
+The idea is to simplify the writing of pulse samples that would usually be declared as incompatible with respect to QUA restrictions for uploading the samples to the OPX. A nice advantage of using this tool is to embed into one single waveform a series of instructions that allows program memory preservation.
+
+Waveform baking is embedded into a new context manager, declared prior to the QUA program, that takes two inputs: 
+
+- the configuration dictionary (the same used to initialize a Quantum Machine instance),
+
+- a padding method: to be chosen between: “right”, “left”, “symmetric_l”, “symmetric_r”.  This string indicates how the should samples be filled up with 0s when they do not correspond to a usual sample (that is if sample length is not a multiple of 4 or if it is shorter than 16 ns). 
+
+    -  “right” setting is the default setting and pads 0s at the end of the baked sample to insert a QUA compatible version in the original configuration file

```

Comment:
“right” setting is the default setting and pads 0s at the end of the baked sample (deleting ending)

Reply:
Maybe we need to change all "sample" to to "samples"? Because a sample is a single point in the configuration, and a waveform is list of samples.

Reply:
and this

Reply:
I think that this is the last comment